coremltools/converters/mil/frontend/torch/ops.py

```diff
@@ -1035,21 +1035,11 @@ def pixel_unshuffle(context, node):
     context.add(perm)
 
 
-def _construct_matmul(x: Var, y: Var, name: Optional[str] = None) -> Var:
```

Comment:
This utils to use `linear` under the certain conditions was an important fix to an existing model,
we cannot remove this 😃